crates/lib/src/jekyll/array.rs

```diff
@@ -195,6 +323,58 @@ impl Filter for ArrayToSentenceStringFilter {
 mod tests {
     use super::*;
 
+    #[test]
+    fn unit_sort_objects() {
+        let input = &liquid_core::value!([
+            {
+                "date": {
+                    "year": 2019,
+                    "month": 3,
+                },
+            },
+            {
+                "date": {
+                    "year": 2018,
+                    "month": 2,
+                },
+            },
+            {
+                "date": {
+                    "year": 2020,
+                    "month": 1,
+                },
+            }
+        ]);
+        let desired_result = liquid_core::value!([
+            {
+                "date": {
+                    "year": 2018,
+                    "month": 2,
+                },
+            },
+            {
+                "date": {
+                    "year": 2019,
+                    "month": 3,
+                },
+            },
+            {
+                "date": {
+                    "year": 2020,
+                    "month": 1,
+                },
+            }
+        ]);
+        assert_eq!(
+            liquid_core::call_filter!(Sort, input, "date.year").unwrap(),
+            desired_result
+        );
+        assert_eq!(
+            liquid_core::call_filter!(Sort, input, "date[\"year\"]").unwrap(),
+            desired_result
+        );
```

Comment:
Ensuring both `object.property` and `object["property"]` work. I could've tested for `object['property']` too, I suppose.